### PR TITLE
Feature/context api

### PR DESCRIPTION
**Pull request by RMartin0717**
Type of change

    [x] New feature
    [ ] Bug Fix
    [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

What does this fix or Change:
Implement Context API

Screen Shot 2021-07-13 at 6 03 50 PM
Screen Shot 2021-07-13 at 6 04 04 PM

Neccesary checkmarks:

    [ ] All Tests are Passing
    [x] The code will run locally

Implements/Fixes/Connected Issues:

    description closes #
    connects issue #15

Testing Changes

    [ ] Please describe what in the world happened

Checklist:

    [x] My code has no unused/commented out code
    [x] My code follows the style guidelines of this project
    [x] I have performed a self-review of my own code
    [ ] I have commented my code, particularly in hard-to-understand areas
    [x] My changes generate no new warnings
    [ ] I have added tests that prove my fix is effective or that my feature works
    [x] New and existing unit tests pass locally with my changes
    [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Components/DarkMode/DarkMode.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from '../../contexts/ThemeContext';
+
+function DarkMode() {
```
Comment (MarcelineBall) [inline src/Components/DarkMode/DarkMode.js:4]: We might want to rename this to make it a bit more clear that it is a button/toggle

File: src/Containers/Doctors/Doctors.js [hunk 2]
```diff
@@ -36,15 +37,18 @@ const Doctors = () => {
...
-    <>
+    <div className='doctors-container' style={{ color: theme.color, background: theme.background }}>
       <h2 className='dr-sub-title'>Doctors here</h2>
       <section>{allDoctors}
       <Link to='/'>
       <button className='home-button'>Home</button>
       </Link>
       </section>
-    </>
+    </div>
```
Comment (MarcelineBall) [inline src/Containers/Doctors/Doctors.js:51]: This looks good for the doctors component. I think we're going to want to do something similar with the MentalHealthPros component for this feature

**Review by MarcelineBall — CHANGES_REQUESTED** [review]
It looks like context got implemented and is working. I think we're just going to want to have it on the MentalHealthPros component before we merge this branch

File: src/Components/DarkMode/DarkMode.js (hunk 1 above)
Comment (battan40) [reply src/Components/DarkMode/DarkMode.js]: When I dive into refactor tonight I can tackle this, if thats ok with everyone.

function DarkModeToggle()

File: src/Components/DarkMode/DarkMode.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  const { toggleDarkMode } = useContext(ThemeContext)
+  return (
+    <>
+      <button onClick={() => toggleDarkMode()}>Dark Mode</button>
```
Comment (battan40) [inline src/Components/DarkMode/DarkMode.js]: This solution is the team work I'm proud to be a part of, simple, minimal.  Good job team.

**Pull request merged**